changelog/std-traits-hasStaticMember.dd

```diff
@@ -0,0 +1,13 @@
+Added `std.traits.hasStaticMember` to check whether a symbol is a static member of a type.
```

Comment:
>I think enforcing that the second line is blank would be good to avoid accidental ambiguity,
>Suggestion: start with a more restrictive policy then relax it as experience develops.

@sprinkle131313 as we seem to have reached a consensus here, would you be so kind to insert an empty line after the title?